src/main/java/io/pakland/mdas/githubstats/domain/Team.java

```diff
@@ -22,6 +22,7 @@ public class Team {
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   private Integer id;
 
+  @Column(name = "slug")
```

Comment:
I think this is not required if the name of the property matches the name of the column.